workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/CreateLicense/index.tsx

```diff
@@ -3,39 +3,186 @@
+		defaultValues: {
+			subscription: undefined,
+		},
+	});
+
+	const {subscription} = watch();
+
+	const stepsInformation: StepsInformation = {
 		[StepCreateLicense.SUBSCRIPTION]: {
 			backStep: StepCreateLicense.SUBSCRIPTION,
-			component: <div>Subscription</div>,
+			component: (
+				<SelectSubscription
+					onSelectSubscription={(subscription: string) => {
+						setValue('subscription', subscription);
+					}}
+					selectedSubscriptionValue={subscription}
+				/>
+			),
 			nextStep: StepCreateLicense.LICENSE_KEY_DETAILS,
 			stepTitle: 'Subscription',
 			title: 'Subscription',
 		},
 		[StepCreateLicense.LICENSE_KEY_DETAILS]: {
 			backStep: StepCreateLicense.SUBSCRIPTION,
-			component: <div>License</div>,
+			component: <LicenseDetails />,
 			nextStep: StepCreateLicense.SUBSCRIPTION,
 			stepTitle: 'License Key Details',
 			title: 'License Key Details',
 		},
 	};
 
+	const ProductCardInfo: ProductCardProps = {
+		licenseKeyData: {
+			endDate: 'Oct 24, 2024',
+			keyType: 'Trial',
+			startDate: 'Sep 24, 2023',
+		},
+		product: {
+			attachments: [],
+			name: {en_US: 'Test Product'},
+			productSpecifications: [],
+			skus: [
+				{
+					price: 0,
+					sku: 'TESTFREEPRODUCTSKU',
+					skuOptions: [],
+				},
+			],
+		},
+		productCreatorAccount: {
+			logoURL: undefined,
+			name: 'Test Name',
+		},
+		userAccount: {
+			emailAddress: 'test@liferay.com',
+		},
+	};
+
+	const ExtendBanner = () => (
+		<>
+			<div className="align-items-center d-flex mb-3 row">
+				<small className="col-6 col-md-4 font-weight-bold m-0">
+					Key type
+				</small>
+				<small className="col-6 col-md-4 subscription-banner-text">
+					{ProductCardInfo.licenseKeyData.keyType}
+				</small>
+			</div>
+
+			<div className="align-items-center d-flex row">
+				<small className="col-6 col-md-4 font-weight-bold m-0">
+					Start Date - Exp. Date
+				</small>
+				<small className="col-6 col-md-4 subscription-banner-text text-nowrap">
+					{ProductCardInfo.licenseKeyData.startDate} &ndash;{' '}
+					{ProductCardInfo.licenseKeyData.endDate}
+				</small>
+			</div>
+		</>
+	);
+
+	const ButtonsInfo = {
+		backButton: {
+			displayType: 'unstyled',
+			show: true,
+		},
+		customizedButton: {
+			displayType: 'secondary',
+			show: step !== StepCreateLicense.SUBSCRIPTION,
+			text: 'Back',
+		},
+		nextButton: {
+			className: 'ml-6',
+			disabled: !subscription,
+			displayType: 'primary',
+			show: true,
+			text: 'Generate Key',
+		},
+	};
+
 	return (
-		<div className="d-flex justify-content-center mb-7">
+		<div className="align-items-center d-flex flex-column">
+			<div className="w-100">
+				<ProductCard
+					ExtendBanner={ExtendBanner}
+					RightSideBanner={() => (
+						<AccountEmailInfo
+							productCreatorAccount={
+								ProductCardInfo.productCreatorAccount
+							}
+							userAccount={ProductCardInfo.userAccount}
+						/>
+					)}
+					creatorAccount={
+						ProductCardInfo.productCreatorAccount as any
```

Comment:
```suggestion
						ProductCardInfo.productCreatorAccount as string
```
Isn't this type a string one?